Telepathy plugin can not be built on openSUSE using 1.2.0-rcgit.26

Trying to build a new version including the telepathy plugins result in an built error. Could be, that openSUSE uses a newer version of dbus? Or did I forget to install a package?
```
[    2s] [26/418] keeping libdbus-1-3-1.8.22-5.1
[    3s] [392/418] keeping libtelepathy-glib0-0.24.1-7.4
[    3s] [396/418] keeping typelib-1_0-TelepathyGlib-0_12-0.24.1-7.4
[    3s] [403/418] keeping telepathy-glib-devel-0.24.1-7.4
...
[    6s] -- Finding suggested package TELEPATHY.
[    6s] --   Disable this using "-DWITH_TELEPATHY=OFF".
[    6s] -- Checking for module 'telepathy-glib'
[    6s] --   Found telepathy-glib, version 0.24.1
[    6s] -- Found TELEPATHY: /usr/lib64/libtelepathy-glib.so  
...
[    8s] cd /home/abuild/rpmbuild/BUILD/Remmina-1.2.0-rcgit.26/build/remmina/po && /usr/bin/msgfmt -o /home/abuild/rpmbuild/BUILD/Remmina-1.2.0-rcgit.26/build/remmina/po/uk.mo /home/abuild/rpmbuild/BUILD/Remmina-1.2.0-rcgit.26/remmina/po/uk.po
[    8s] CMakeFiles/remmina-plugin-telepathy.dir/telepathy_handler.c.o: In function `remmina_tp_handler_register':
[    8s] /home/abuild/rpmbuild/BUILD/Remmina-1.2.0-rcgit.26/remmina-plugins/telepathy/telepathy_handler.c:108: undefined reference to `dbus_g_connection_register_g_object'
[    8s] CMakeFiles/remmina-plugin-telepathy.dir/telepathy_channel_handler.c.o: In function `remmina_tp_channel_handler_on_response':
[    8s] /home/abuild/rpmbuild/BUILD/Remmina-1.2.0-rcgit.26/remmina-plugins/telepathy/telepathy_channel_handler.c:203: undefined reference to `dbus_g_method_return_error'
[    8s] CMakeFiles/remmina-plugin-telepathy.dir/telepathy_channel_handler.c.o: In function `tp_svc_client_handler_return_from_handle_channels':
[    8s] /usr/include/telepathy-1.0/telepathy-glib/_gen/tp-svc-client.h:86: undefined reference to `dbus_g_method_return'
[    8s] CMakeFiles/remmina-plugin-telepathy.dir/telepathy_channel_handler.c.o: In function `remmina_tp_channel_handler_accept':
[    8s] /home/abuild/rpmbuild/BUILD/Remmina-1.2.0-rcgit.26/remmina-plugins/telepathy/telepathy_channel_handler.c:181: undefined reference to `dbus_g_type_struct_get'
[    8s] collect2: error: ld returned 1 exit status
[    8s] remmina-plugins/telepathy/CMakeFiles/remmina-plugin-telepathy.dir/build.make:161: recipe for target 'remmina-plugins/telepathy/remmina-plugin-telepathy.so' failed
[    8s] make[2]: *** [remmina-plugins/telepathy/remmina-plugin-telepathy.so] Error 1

--- a/plugins/telepathy/CMakeLists.txt
+++ b/plugins/telepathy/CMakeLists.txt
@@ -43,10 +43,11 @@ set_target_properties(remmina-plugin-telepathy PROPERTIES PREFIX "")
 set_target_properties(remmina-plugin-telepathy PROPERTIES NO_SONAME 1)
 
 find_required_package(GTK3)
+pkg_check_modules(PC_DBUS_GLIB dbus-glib-1)
 
 include_directories(${REMMINA_COMMON_INCLUDE_DIRS} ${TELEPATHY_INCLUDE_DIRS})
 target_link_libraries(remmina-plugin-telepathy ${REMMINA_COMMON_LIBRARIES}
-    ${TELEPATHY_LIBRARIES})
+    ${TELEPATHY_LIBRARIES} ${PC_DBUS_GLIB_LIBRARIES})
 
 install(TARGETS remmina-plugin-telepathy DESTINATION ${REMMINA_PLUGINDIR})
 

